Prebuilt binaries aren't available for v7.1.0 release

[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT share sensitive information, whether personal, proprietary, or otherwise! )

## Expected Behavior

When installing the module, a pre-built binary can be downloaded instead of compiling the code

[NOTE]: # ( Tell us what you expected to happen. )

## [Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results

[NOTE]: # ( Provide any other relevant log data. )
[TIP]:  # ( Scrub logs and diagnostic information for sensitive information )

A successful download from v7.0.2 looks like:
```
> /usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js clean configure
gyp WARN install got an error, rolling back install
gyp ERR! configure error
gyp ERR! stack Error: There was a fatal problem while downloading/extracting the tarball
gyp ERR! stack at afterTarball (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/install.js:212:21)
gyp ERR! System Linux 5.8.0-1035-aws
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "clean" "configure"
gyp ERR! cwd /app/apps/contentweb/node_modules/@newrelic/native-metrics
gyp ERR! node -v v14.17.6
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
install successful: _newrelic_native_metrics-7_0_2-native_metrics-83-linux-x64
```

An unsuccessful download from v7.1.0 looks like:
```
> /usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js clean configure
  gyp WARN install got an error, rolling back install
  gyp ERR! configure error
  gyp ERR! stack Error: There was a fatal problem while downloading/extracting the tarball
  gyp ERR! stack     at afterTarball (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/install.js:212:21)
  gyp ERR! System Linux 5.8.0-1035-aws
  gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "clean" "configure"
  gyp ERR! cwd /app/apps/respweb/node_modules/@newrelic/native-metrics
  gyp ERR! node -v v14.17.6
  gyp ERR! node-gyp -v v5.1.0
  gyp ERR! not ok
  Failed to execute native-metrics install: No pre-built artifacts for your OS/architecture.
  npm ERR! code ELIFECYCLE
  npm ERR! errno 1
  npm ERR! @newrelic/native-metrics@7.1.0 install: `node ./lib/pre-build.js install native_metrics`
  npm ERR! Exit status 1
  npm ERR!
  npm ERR! Failed at the @newrelic/native-metrics@7.1.0 install script.
  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

## Steps to Reproduce

[NOTE]: # ( Please be as specific as possible. )
[TIP]:  # ( Link a sample application that demonstrates the issue. )
npm install @newrelic/native-metrics@7.1.0

## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )


* Node version:14.17.6
* Operating System and version: Built inside a docker instance based off of `ubuntu:focal`

## Additional context

[TIP]:  # ( Add any other context about the problem here. For example, relevant community posts or support tickets. )

--- a/.github/workflows/release-workflow.yml
+++ b/.github/workflows/release-workflow.yml
@@ -2,12 +2,15 @@ name: native-metrics Binary Upload
 
 on:
   workflow_dispatch:
-  push:
-    tags:
-      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
+  workflow_run:
+    workflows: ["Create Release"]
+    types:
+      - completed
 
 jobs:
   build-test-upload:
+     # Check if this was a manual invocation(workflow_dispatch) or triggered(workflow_run) and successful
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event.workflow_run && github.event.workflow_run.conclusion == 'success' }}
     runs-on: ubuntu-latest
 
     strategy:

